xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -419,6 +455,8 @@ namespace PVR
     mutable XbmcThreads::EndTime m_recordingSizeRefetchTimeout;
     int64_t m_sizeInBytes = 0; /*!< the size of the recording in bytes */
     bool m_bDirty = false;
+    int m_iClientProviderUniqueId =
+        PVR_PROVIDER_INVALID_UID; /*!< provider uid associated with this recording on the client */
```

Comment:
We should not follow the bad design of this class to make members public!

Reply:
Well, wrong line. My note was referring to `std::string m_strProviderName; /*!< name of the provider this recording is from */`

Reply:
Gotcha, will change that.

Reply:
Done